common/lib/xmodule/xmodule/js/src/capa/display.js

```diff
@@ -659,6 +659,7 @@
                         that.custom_alert("Answer submitted successfully");
                     }
 
+                    fetchPrePostAssessmentsScore()
```

Comment:
Let's document this core change

Reply:
We will document this once this will be merged in faraz/LP-1611 story.